components/camel-mongodb/src/main/java/org/apache/camel/component/mongodb/converters/MongoDbBasicConverters.java

```diff
@@ -119,6 +119,9 @@ private static boolean isBson(byte[] input) {
 
     @Converter
```

Comment:
We need to make @Converter(allowNull = true) as the converter can return null as a valid response

Reply:
In this case this should be considered be a _miss_. According the [documentation](https://camel.apache.org/manual/latest/type-converter.html#TypeConverter-Returningnullvalues), this would be the way to handle.

Reply:
Yes can you add that fix

Reply:
I did not explain myself properly, sorry. I was meaning that according the documentation, when the converted value is expected to be a _miss_ then `null` should be returned. If `null` is allowed to be a valid response, then, we should include the annotation as you said. In this case, `null` won't be a valid response, but just a _miss_, so I understand that the annotation is not needed. At least, this is what I understood from the doc, or am I misunderstanding something?

Reply:
Yes you misunderstod. A miss is regarded as a from -> to combo that should not be attempted anymore in the future (miss).

Reply:
Thanks for clarifying. I've just pushed the change.